samples/msgext-ai-sentiment-analysis/nodejs/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-msgext-ai-sentiment-analysis-nodejs",
+    "source": "officeDev",
+    "title": "Sentiment analysis with messaging extension and Azure Open AI",
```

Comment:
What sentiment analysis?
Add like below:
Sentiment analysis of Teams chat message using messaging extension and Azure Open AI

Reply:
fixed